incubator/elasticsearch/values.yaml

```diff
@@ -50,8 +53,11 @@ data:
   name: data
   replicas: 2
   heapSize: "1536m"
-  storage: "30Gi"
-  # storageClass: "ssd"
+  persistence:
+    enabled: true
+    accessMode: ReadWriteOnce
+    storage: "30Gi"
```

Comment:
Please rename to `size`, only for consistency w/ other charts.

Reply:
WIll update to size.